### DISCUSSION TITLE
Change Temperature to Farenheit

### DISCUSSION THREAD

**Discussion by davewalder** [saso5/saso5.github.io#2] [General]
Hello, I just started using your LYWSD02 Clock sync program and it works great.  I saw that the option to change the temperature from C to F was commented out.  Does this work if the comments are removed?

Thank you for your work.

**Comment by saso5** [comment]
No, I never got it to work. As mentioned on the page, the code is based on https://github.com/h4/lywsd02/blob/master/lywsd02/client.py, but I never got that lib to work (I use windows).   

I can read the value, but setting the value does no do anything. If you figure it out please let me know how :)
